conjure-java-core/src/integrationInput/java/com/palantir/insensitive/InsensitiveEnum.java

```diff
@@ -75,7 +74,6 @@ public static InsensitiveEnum valueOf(String value) {
             case "ONE_HUNDRED":
                 return ONE_HUNDRED;
             default:
-                ConjureEnums.validate(value);
                 return new InsensitiveEnum(Value.UNKNOWN, value);
```

Comment:
You can see this matches the pre 2.8.0 behaviour:

https://github.com/palantir/conjure-java/blob/2.7.1/conjure-java-core/src/integrationInput/java/com/palantir/product/EnumExample.java#L62-L76